morphic.js

```diff
@@ -7682,19 +7682,21 @@ MenuMorph.prototype.unselectAllItems = function () {
     this.changed();
 };
 
-MenuMorph.prototype.popup = function (world, pos) {
+MenuMorph.prototype.popup = function (world, pos, submenu) {
```

Comment:
Comment added.  No idea what you have against Boolean args.  Okay, not if there are five of them in one procedure.